ad_block_client_wrap.cc

```diff
@@ -304,6 +306,21 @@ void AdBlockClientWrap::GetParsingStats(
   args.GetReturnValue().Set(stats);
 }
 
+void AdBlockClientWrap::GetFingerprint(
+    const FunctionCallbackInfo<Value>& args) {
+  Isolate* isolate = args.GetIsolate();
+  AdBlockClientWrap* obj =
+    ObjectWrap::Unwrap<AdBlockClientWrap>(args.Holder());
+  String::Utf8Value str(args[0]->ToString());
+  const char * inputBuffer = *str;
+
+  char * fingerprintBuffer = new char[AdBlockClient::kFingerprintSize + 1];
+  if (obj->getFingerprint(fingerprintBuffer, inputBuffer)) {
+    args.GetReturnValue().Set(String::NewFromUtf8(isolate, fingerprintBuffer));
+    delete[] fingerprintBuffer;
```

Comment:
You have to call `delete[] fingerprintBuffer;` outside of if scope

Reply:
good catch